### ISSUE TITLE
Sequence contains no elements

### ISSUE THREAD

**Issue by PerusPaco** [lay295/TwitchDownloader#614]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Edition

Unsure

### Describe your issue here

Hello,

I have this issue in which any VOD I try to download from the channel https://www.twitch.tv/creativemonkeyz is getting an error and I cannot download it. The error is "Sequence contains no elements"
Basically I cannot even select the size of the video (1080p/720p, etc) and it's ONLY for this channel. I've tried with other twitch channel and it's working.
Could you please support?

Thank you!

**Comment by ScrubN** [comment]
You need to subscribe and provide the OAuth from your account. 
![image](https://user-images.githubusercontent.com/72096833/225725795-17d200b6-90fb-4512-8faa-d19765aa26b3.png)

**Comment by PerusPaco** [comment]
but there is no way do download it without subscription? :(

**Comment by ScrubN** [comment]
No. Our current API requests require OAuth for subscriber VODs and there are no plans to find workarounds. That would technically be piracy which could result in legal problems.

**Comment by PerusPaco** [comment]
fair. thanks for support!

**Comment by Varga-Levente** [comment]
Same issue here but the vod is public

**Comment by ScrubN** [comment]
> Same issue here but the vod is public

Looks like your vod doesn't work on the website either.
![image](https://github.com/lay295/TwitchDownloader/assets/72096833/b564b102-9aed-4b2d-bbb6-3248e61b2141)

The API request returns forbidden and every video quality is marked as restricted. My guess is that the VOD is indeed sub-only but Twitch broke the website again so you it doesn't say that it's sub only.
![image](https://github.com/lay295/TwitchDownloader/assets/72096833/d147b90b-726e-4049-b77f-2ae67f5d6f79)

**Comment by ScrubN** [comment]
> but Twitch broke the website again so you it doesn't say that it's sub only.

It looks like Twitch might be trying out a new form of sub-only VODs? The video page has a unique *Subscribe to watch* button that isn't on any other sub-only videos:
![image](https://github.com/lay295/TwitchDownloader/assets/72096833/d56ac0e8-7ddd-4232-9ea8-128a439add12)

And other sub-only videos don't return forbidden in the API response (though the video qualities are still restricted):
![image](https://github.com/lay295/TwitchDownloader/assets/72096833/029a2d8e-5757-4984-870a-b062f5249d4d)
